controllers/authController.js

```diff
@@ -0,0 +1,64 @@
+import jwt from 'jsonwebtoken';
+import AppError from '../errors/AppError';
+import * as UserModel from '../models/UserModel';
```

Comment:
You import 8 functions and only use 3. Please only import methods that you will use.